AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5575,6 +5575,7 @@ sport1.de#%#AG_onLoad(function() {if (!document.cookie.includes("euconsent-v2"))
 roastmarket.de,heute.at#%#AG_onLoad(function(){if(!document.cookie.includes("eupubconsent-v2")){var a=new MutationObserver(function(){var b=document.querySelector("#onetrust-accept-btn-handler");b&&(a.disconnect(),b.click())});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)}});
 thehouseofportable.com#%#(function() { try { if ("undefined" != typeof localStorage) { localStorage.setItem('accepted', '1') } } catch (ex) {} })();
 coolmath.com#%#//scriptlet('set-cookie', 'GDPR_Reject', 'true')
+24ur.com#%#AG_onLoad(function() {setTimeout(function() {var a = document.querySelector('.callout__content > a.callout__button');a&&a.click();},1000)})
```

Comment:
this rule doesn't work at the main page (because there is no videos on it)
btw I think it's better to use mutationObserver to avoid blinking